getMultiStarts.m

```diff
@@ -238,7 +238,16 @@
         % different initialization
         if (strcmp(options.localOptimizer, 'fmincon'))
             if (strcmp(options.localOptimizerOptions.Hessian, 'on'))
-                [J_0,~,~] = negLogPostWErrorCount(parameters.MS.par0(:,i)); % objectiveWrapWErrorCount(parameters.MS.par0(:,i),objective_function,options.obj_type,options.objOutNumber);
+                % We have to check Hessian AND gradient computation, since
```

Comment:
evaluating twice negLogPostWErrorCount is not necessary and for large models compuationally demanding and should thus be avoided.
If `options.localOptimizerOptions.Hessian` is 'on' do we ever call `[J_1,~] = negLogPostWErrorCount(...)`?